gitattributes.go

```diff
@@ -0,0 +1,251 @@
+package enry
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"regexp"
+	"strings"
+
+	"gopkg.in/src-d/enry.v1/data"
+)
+
+type attrType int
+
+const (
+	vendor attrType = iota
+	documentation
+	generated
+	language
+)
+
+const _attrType_name = "vendordocumentationgeneratedlanguage"
```

Comment:
What are indicating the underscores? I though this is not so idiomatic in Go.

On the other hand, why don't you use the previous constants as a string turning attrType a string type? Seems like it's adding unnecessary complexity, with _attr_index and the method for attrType String